### PR TITLE
.Net: Upgrade to v1.0.1

### PR DESCRIPTION
**Pull request by dmytrostruk**
### Motivation and Context

Upgraded packages to `v1.0.1`. Some of the packages will have suffix `alpha` or `preview`.

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

**Review by lukins-cz — CHANGES_REQUESTED** [review]
🤔

**Comment by stephentoub** [comment]
1.0.0 (not 1.0.0)?

**Review by lukins-cz — COMMENTED** [review]
🤔

**Comment by dmytrostruk** [comment]
> 1.0.1 (not 1.0.0)?

@stephentoub We are going to keep some of the projects in pre-release state with `alpha` or `preview` suffix in package version. Taking into account that current version of all our packages is `1.0.0-rc4`, we can't use `1.0.0-alpha` or `1.0.0-preview` anymore, because version `1.0.0-rc4` will be resolved as the latest version, which won't be a case. 

We decided to start with `1.0.1` version to apply `alpha` and `preview` suffixes for some of the packages without versioning problems.

**Review by SergeyMenshykh — APPROVED** [review]

File: dotnet/src/Experimental/Assistants/Experimental.Assistants.csproj [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <AssemblyName>Microsoft.SemanticKernel.Experimental.Assistants</AssemblyName>
     <RootNamespace>Microsoft.SemanticKernel.Experimental.Assistants</RootNamespace>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <VersionSuffix>alpha</VersionSuffix>
```
Comment (markwallace-microsoft) [inline dotnet/src/Experimental/Assistants/Experimental.Assistants.csproj:7]: This project is no longer in the solution and will be deleted soon. 

There is a new Experimental.Agents project to replace this, can you update that one instead.

**Review by markwallace-microsoft — COMMENTED** [review]
One change required

File: dotnet/src/Experimental/Assistants/Experimental.Assistants.csproj (hunk 1 above)
Comment (dmytrostruk) [reply dotnet/src/Experimental/Assistants/Experimental.Assistants.csproj:7]: Good catch, thanks. Addressed!

**Review by markwallace-microsoft — APPROVED** [review]

**Pull request merged**